cfme/tests/v2v/test_rest_migrations.py

```diff
@@ -0,0 +1,126 @@
+"""Test to check v2v migration rest API"""
+import fauxfactory
+import pytest
+
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.markers.env_markers.provider import ONE_PER_VERSION, ONE_PER_TYPE
+
+
+pytestmark = [
+    pytest.mark.provider(
+        classes=[RHEVMProvider],
+        selector=ONE_PER_VERSION,
+        required_flags=["v2v"]
+    ),
+    pytest.mark.provider(
+        classes=[VMwareProvider],
+        selector=ONE_PER_TYPE,
+        fixture_name="second_provider",
+        required_flags=["v2v"]
+    )
+]
+
+
+@pytest.fixture(scope="function")
+def get_clusters(appliance, v2v_providers):
+    clusters = {}
+    source_cluster = v2v_providers.vmware_provider.data.get("clusters")[0]
+    target_cluster = v2v_providers.rhv_provider.data.get("clusters")[0]
+    cluster_db = {
+        cluster.name: cluster for cluster in appliance.rest_api.collections.clusters.all
+    }
+
+    if source_cluster in cluster_db.keys():
+        clusters["source"] = cluster_db[source_cluster].href
+    if target_cluster in cluster_db.keys():
```

Comment:
Did not added `else` since wanted to check both `source` and `target` things; also did not add `elif` since if first condition passed, it will not check second condition which is not we wanted.

Updated try/expect/skip there, let me know if you feel this is too much (which I believe) so will wrap two try/expect into one.

Reply:
@mshriver did not added `else` since wanted to check both `source` and `target` things; also did not add `elif` since if first condition passed, it will not check second condition which is not we wanted.

Updated try/expect/skip there, let me know if you feel this is too much (which I believe) so will wrap two try/expect into one.